instrumentation/ktor/ktor-2.0/testing/src/main/kotlin/io/opentelemetry/instrumentation/ktor/v2_0/client/AbstractKtorHttpClientTest.kt

```diff
@@ -71,4 +77,24 @@ abstract class AbstractKtorHttpClientTest : AbstractHttpClientTest<HttpRequestBu
       }
     }
   }
+
+  @Test
+  fun checkSpanEndsAfterBodyReceived() {
+    val method = "GET"
+    val path = "/long-request"
+    val uri = resolveAddress(path)
+    val responseCode = doRequest(method, uri)
+
+    Assertions.assertThat(responseCode).isEqualTo(200)
+
+    testing.waitAndAssertTraces(
+      Consumer { trace: TraceAssert ->
+        val span = trace.getSpan(0)
+        OpenTelemetryAssertions.assertThat(span).hasKind(SpanKind.CLIENT)
+        Assertions.assertThat(span.endEpochNanos - span.startEpochNanos >= 1000000)
```

Comment:
we typically static import `Assertions` and `OpenTelemetryAssertions` (https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/docs/contributing/style-guideline.md#static-imports)